packages/lib-classifier/src/components/Classifier/components/Layout/Layout.js

```diff
@@ -1,20 +1,27 @@
 import { observer } from 'mobx-react'
-
 import { useStores } from '@hooks'
 import getLayout from './helpers/getLayout'
 
 function storeMapper(classifierStore) {
   const workflow = classifierStore.workflows.active
+  const subject = classifierStore.subjects.active
 
   return {
-    layout: workflow?.layout
+    multiFrameSubject: subject?.isMultiFrameSubject,
+    usesTranscriptionTask: workflow?.usesTranscriptionTask
   }
 }
 
-
 function Layout() {
-  // `getLayout()` will always return the default layout as a fallback
-  const { layout } = useStores(storeMapper)
+  const { multiFrameSubject, usesTranscriptionTask } = useStores(storeMapper)
+
+  let layout = 'default'
+  if (usesTranscriptionTask) {
+    layout = 'noMaxWidth'
+  } else if (multiFrameSubject && !usesTranscriptionTask) {
+    layout = 'multiFrame'
```

Comment:
Reading through the subject code, this will set the layout to `multiFrame` for both PH-TESS (2 subject locations) and SuperWASP Black Hole Hunters (4 subject locations.)


I don’t think that’s the intended layout for those projects.

Reply:
Here's a screenshot of PH-TESS with the component inspector showing that it now uses the `MultiFrame` layout. It looks ok in the classifier, but I'm not sure that this is intentional. We probably want a test somewhere that will catch this.

<img width="1226" alt="PH-TESS in the dev classifier, with the component inspector open to show that it now uses the MultiFrame layout component." src="https://user-images.githubusercontent.com/59547/221812030-449dfc2a-b7a1-479a-afbb-f5612de0738d.png">

Reply:
If a non-static layout is not a blocker for this PR, then I agree the non-default layout component does not need to be used for TESS and SuperWASP. We can definitely check for subjects that have multiple locations that are _only_ images. The Subject.js store file already does this in `get viewer()`.

Reply:
Reflecting on the subject with `getType` might work too eg. `if (getType(subject) === ImageSubject)`.

Reply:
SuperWASP subjects have up to 3 images but I think the type that it uses is `SingleJSONSubject`. 

PH-TESS uses `ImageAndTextSubject` due to a bug in the Python client, which uploads `.json` files as `.txt`.

Reply:
On second thoughts, `getType(subject)` won't be useful to distinguish between subjects. We use `types.refinement` to define subject types, in which case `getType(subject)` will return the base `SubjectResource` type. See https://github.com/mobxjs/mobx-state-tree/issues/1369.

That's why the subject tests can't distinguish between different types of subjects eg. the multi-image subject here is the same type as a single image subject or a JSON data subject.
https://github.com/zooniverse/front-end-monorepo/blob/dabf1b8254771919c429f844c5e7aef50e3de0fb/packages/lib-classifier/src/store/SubjectStore/SubjectType/SubjectType.spec.js#L73-L96

That's a shame, because subject types seem like the ideal way to distinguish between types of subject.

Reply:
I've opened #4320 so that `getType(subject).name` will return a specific name for each type of subject eg. `SingleImageSubject`, `ImageSubject`, `SingleJSONSubject` etc. That might work here as a more specific test of the type of subject.